test/unittests/mongoBackend/mongoSubscribeContext_test.cpp

```diff
@@ -1100,17 +1083,13 @@ TEST(mongoSubscribeContext, EntN_Attr0_CN)
 
     std::vector<BSONElement> conds = sub.getField("conditions").Array();
     std::vector<BSONElement> condValues;
-    ASSERT_EQ(2, conds.size());
+    ASSERT_EQ(1, conds.size());
```

Comment:
Another interesting consequence of this refactoring is that now it "collapse" all attributes in no matter how much condition vectors element into just one. I mean, previous to this PR if we have in the API:

conditions: [
{ type: ONCHANGE, values: [ A, B ] },
{ type: ONCHANGE, values: [ C, D ] },
]

that ended in DB in a similar estructure.

Now, with this PR, that will be stored in DB as:

conditions: [ 
{type: ONCHANGE, value: [ A, B, C, D ]
]

I think this is benefitial, in some way a step in the road to https://github.com/telefonicaid/fiware-orion/issues/1851